test/quantization/test_quantize_fx.py

```diff
@@ -406,6 +406,122 @@ def forward(self, x):
         FileCheck().check_count('FakeQuantize = prim::GetAttr[name="activation_post_process', 4, exactly=True) \
                    .run(scripted.graph)
 
+    @skipIfNoFBGEMM
+    def test_custom_module_class(self):
+        class CustomModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(1, 1, 1)
+
+            def forward(self, x):
+                return self.conv(x)
+
+        class ObservedCustomModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, x):
+                pass
+
+            @classmethod
+            def from_float(cls, float_module):
+                assert hasattr(float_module, 'qconfig')
+                float_module._FLOAT_MODULE = CustomModule
+                return float_module
+
+        class QuantizedCustomModule(torch.nn.Module):
+            def __init__(self, conv):
+                super().__init__()
+                self.conv = conv
+
+            def forward(self, x):
+                return self.conv(x)
+
+            @classmethod
+            def from_observed(cls, observed_module):
+                assert hasattr(observed_module, 'qconfig')
+                assert hasattr(observed_module, 'activation_post_process')
+                observed_module.conv.activation_post_process = \
```

Comment:
Why does this have to be set by the user? Can't it be done using the qconfig?

Reply:
observer_module.activation_post_process is initialized from qconfig

Reply:
this should be considered using observer_module.activation_post_process to quantize the module, similar as we use mod.activation_post_process in other quantized modules like: https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/conv.py#L256